src/index.js

```diff
@@ -1,8 +1,65 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import App from './containers/App';
+import { FILTERS } from './constants/constants';
+import { CREATE_BOOK, REMOVE_BOOK } from './actions/index';
+import category from './reducers/categories';
+
+const initialState = [
+  {
+    title: 'A Hundred Years of Solitude',
+    category: FILTERS.biography,
+    id: Math.round(Math.random() * 10000),
```

Comment:
* [ ] Math.random generates random number not unique numbers, for a robust unique number generation, we can use libraries like `shortid` or `uuid` these packages are built purposely for that. let's tweak the code a bit.